code/datums/mind.dm

```diff
@@ -1233,6 +1233,9 @@
 				var/datum/syndicate_contract/new_contract = new(H, src, list(), target)
 				new_contract.reward_tc = list(0, 0, 0)
 				H.contracts += new_contract
+				for(var/difficulty in EXTRACTION_DIFFICULTY_EASY to EXTRACTION_DIFFICULTY_HARD)
+					var/amount_tc = H.calculate_tc_reward(length(H.contracts), difficulty)
+					new_contract.reward_tc[difficulty] = amount_tc
```

Comment:
shouldnt this also do `new_contract.total_earnable_tc[difficulty] += amount_tc`?

Reply:
`total_earnable` is only used/defined in the `generate_contracts` proc to even out the number of TC per difficulty. I didn't want to mess with the rewards of preexisting contracts.